privacyidea/lib/eventhandler/counterhandler.py

```diff
@@ -99,14 +99,12 @@ def do(self, action, options=None):
         counter_name = handler_options.get("counter_name")
 
         if action == "increase_counter":
-            r = increase(counter_name)
-            log.debug(u"Increased the counter {0!s} to {1!s}.".format(counter_name,
-                                                                      r))
+            increase(counter_name)
+            log.debug(u"Increased the counter {0!s}.".format(counter_name))
```

Comment:
We could have also returned the overall new counter value (over all nodes). This would have kept the log entry the same. But it is not that important to me.

Reply:
Yes, I agree that would be nice. But this results in another ``SELECT`` query -- which, I think, would be totally fine if the result is used for something important. But as it was only written to the debug log, I thought we could save one query and speed up things a bit :)